bigquery_etl/glam/generate.py

```diff
@@ -78,9 +78,9 @@ def main():
     if not dataset_path.is_dir():
         raise NotADirectoryError(f"path to {dataset_path} not found")
 
-    build_date_udf_mapping = dict(
-        org_mozilla_fenix="`moz-fx-data-shared-prod`.udf.fenix_build_to_datetime"
-    )
+    build_date_udf_mapping = {
+        args.prefix: "`moz-fx-data-shared-prod`.udf.fenix_build_to_datetime"
```

Comment:
If the product is not fenix, this is the wrong udf to use. If this returns null by default though, it should be fine. This can be fixed easily if broken since it's not used in the core aggregation.

Reply:
Oh that's true.  I'll add the the specific ids